backend/.env.template

```diff
@@ -42,6 +42,7 @@ EMAIL_SMTP_URL=$EMAIL_SMTP_URL
 EMAIL_SMTP_PORT=587
 EMAIL_LINK_VERIFICATION=$EMAIL_LINK_VERIFICATION
 EMAIL_LINK_SETPASSWORD=$EMAIL_LINK_SETPASSWORD
+EMAIL_LINK_FORGOTPASSWORD=$EMAIL_LINK_FORGOTPASSWORD
```

Comment:
We need this in the `deployment/bare_metal/.env.dist`
Please update the config version in `backend(src/config/index.ts`

Reply:
done 👍🏼